pkg/didcomm/crypto/crypter.go

```diff
@@ -30,9 +35,9 @@ type Crypter interface {
 // KeyPair represents a private/public key pair each with 32 bytes in size
 type KeyPair struct {
 	// Priv is a private key
-	Priv *[chacha.KeySize]byte
+	Priv *[chacha.KeySize]byte `json:"private"`
```

Comment:
It's a bit odd to define a KeySize const in this file and not use it?

Reply:
i.e., you are still importing chacha into this interface package.

Reply:
done remove all chacha.KeySize